backend/application/syson-application/src/test/java/org/eclipse/syson/application/export/ImportExportTests.java

```diff
@@ -872,6 +872,53 @@ public void checkOccurrenceTest() throws IOException {
         this.checker.check(input, expected);
     }
 
+    @Test
+    @DisplayName("GIVEN a model with a reference to an attribute with a short name only, WHEN importing and exporting the model, THEN the reference should be able to use the shortName as identifier")
+    public void checkReferencingWithShortName() throws IOException {
+        var input = """
+                part p1 {
+                    attribute <attr1>;
+                }
+                part p2 :> p1 {
+                    attribute :>> attr1;
+                }""";
+        this.checker.check(input, input);
+    }
+
+    @Test
+    @DisplayName("GIVEN a model with a references to elements, WHEN importing and exporting the model, THEN the references should privileged short name over declared name when possible")
+    public void checkReferencingIdentifiers() throws IOException {
+        var input = """
+                enum def Enum1 {
+                    <e1> enumeration1;
+                    enumeration2;
+                }
+                part part1 {
+                    attribute attribute1 : Enum1 = Enum1::enumeration1;
+                    attribute attribute2 : Enum1 = enumeration1;
+                    attribute attribute3 : Enum1 = e1;
+                    attribute attribute4 : Enum1 = Enum1::enumeration2;
+                    attribute attribute5 = Enum1::enumeration1;
+                }""";
+        // Attribute 1,2,3 and 4 are type with Enum1. It gives them direct access to enumeration literals through the featureTyping. In this case use the shortName if available (not the case for
+        // attribute4)
```

Comment:
// attribute4) => it seems this line could be put after "not the case for" just above

Reply:
Are you sure if I put it on the same line I get a line lenght of 207 char. I thought that the limit was 200.